sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -38,8 +38,14 @@ public partial class EventHubOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsF
     public partial class InitialOffsetOptions
     {
         public InitialOffsetOptions() { }
-        public string EnqueuedTimeUTC { get { throw null; } set { } }
-        public string Type { get { throw null; } set { } }
+        public System.DateTimeOffset? EnqueuedTimeUtc { get { throw null; } set { } }
+        public Microsoft.Azure.WebJobs.EventHubs.OffsetType? Type { get { throw null; } set { } }
```

Comment:
@sidkri we are updating the Type property to be an enum, and the EnqueuedTimeUtc property to be a DateTimeOffset. Previously, these were both strings. Do you know if this will cause any issues for any customers?

Reply:
My original logic in EventHubWebJobsBuilderExtensions.ConfigureInitialOffsetOptions() allows configuring a timestamp with no timezone specified (the timezone of the local machine is assumed and converted to UTC). Not sure if that is still supported here via DateTimeOffset?  Otherwise the change is good.

Reply:
Yes this is still supported.